src/tests/ftest/util/control_test_base.py

```diff
@@ -28,6 +28,29 @@
 from command_utils import CommandFailure
 
 
+def cleanup_output(output):
+    """Cleanup output that is of this form:
+        [(host, "", ""), ("", "data1", "data2"), ("", "data1", "data2")]
+
+    Args:
+        output (list): output to be parsed.
+
+    Returns:
+        dict: integrated dictionary containing information for each host.
+
+    """
+    host = None
+    info = {}
```

Comment:
If you use defaultdict, you don't need to check for key error.
```
from collections import defaultdict
info = defaultdict(list)
info[host].append(item[1:])
```

Reply:
The reason I'm checking for `item[0]` is because the data comes in thist format from the regex:
`[(host, "", ""), ("", "data1", "data2"), ("", "data1", "data2")]`
so I'm checking for the first element to record the hostname and then I append the information to the pertaining hostname.

Reply:
Ah, I see what you mean. That makes sense, I'll implement that.